### ISSUE TITLE
Uncaught ReferenceError: process is not defined

### ISSUE THREAD

**Issue by amourzenkov-sc** [substantial/updeep#117]
When running a project which has indirect **updeep** dependency I get the following error in a browser:

![image](https://user-images.githubusercontent.com/24649844/56193745-0b920780-603a-11e9-8561-52dff2cb883d.png)

The browser has problems with **lib/freeze.js line 32**

```javascript
if (process.env.UPDEEP_MODE === 'dangerously_never_freeze') {
```

I think it should be replaced with

```javascript
if (process && process.env && process.env.UPDEEP_MODE === 'dangerously_never_freeze') {
```

to make `UPDEEP_MODE` environment variable truly optional.

**Comment by arturparkhisenko** [comment]
Solutions while PR is open:
- Provide process ([example](https://stackoverflow.com/a/64553486/1059640))
- Use any [alternative](https://www.npmjs.com/search?q=deep%20merge)

**Comment by aaronjensen** [comment]
Fixed in fb4f674
